dldashboard/web/src/components/Memory/MemoryDetail.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
```

Comment:
Can we write component level documentation for these ?
For example why do we need memoryManager , MemoryDetail and MemoryList and MemoryVirtualizedTable and how they serve each other ?

Reply:
@kavyasrinet Yep that makes sense. Do you want me to add comments in the code, or is there something external? LMK if there is a template I should follow. Thanks!

Reply:
Comments in the code or function level doctrings should help.

Reply:
Hi @kavyasrinet, I added the documentation. Let me know if it looks ok?

Reply:
Thanks ! Taking a look now